wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -697,6 +702,8 @@ func EvalCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (sstore.
 	newPk, rtnErr := EvalMetaCommand(ctxWithHistory, pk)
 	if rtnErr == nil {
 		update, rtnErr = HandleCommand(ctxWithHistory, newPk)
+	} else {
```

Comment:
bug fix - I found a bug where if I ran `/copyfile ~/'` there would be a nil ptr dereference because metacmd parsing fails and newPk here ends up as nil. rtnErr isn't properly checked here, so I fixed this by adding a return if there is an err here